src/main/java/core/basesyntax/Track.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Track extends Machine {
+    public void setTrackOne(Machine trackOne) {
+    }
```

Comment:
Could you explain what does this method?

Reply:
Still trying to understand oop, apparently, I misunderstood constructor and setter